server/services/owntracks/lib/handleMqttMessage.js

```diff
@@ -0,0 +1,46 @@
+/* eslint dot-notation: [2, {"allowPattern": "(_[a-z]+)+$"}] */
+
+const logger = require('../../../utils/logger');
+
+/**
+ * @description Handle a new message receive in MQTT.
+ * @param {string} topic - MQTT topic.
+ * @param {Object} message - The message sent.
+ * @returns {undefined}
+ * @example
+ * handleMqttMessage('owntracks/location/deviceId', data);
+ */
+function handleMqttMessage(topic, message) {
+  const splittedTopic = topic.split('/');
+  const user = splittedTopic[2];
```

Comment:
So the user needs to put the user selector in the deviceId ? Why not in the userId? It would make more sense no?